Gems/ScriptAutomation/Code/Source/ScriptAutomationSystemComponent.cpp

```diff
@@ -326,34 +326,40 @@ namespace AZ::ScriptAutomation
     void ScriptAutomationSystemComponent::ExecuteScript(const char* scriptFilePath)
     {
         AZ::Data::Asset<AZ::ScriptAsset> scriptAsset = LoadScriptAssetFromPath(scriptFilePath, *m_scriptContext.get());
-        [[maybe_unused]] AZStd::string localScriptFilePath = scriptFilePath;
+        AZStd::string localScriptFilePath = scriptFilePath;
```

Comment:
weird, we are removing `[[maybe_unused]]` but adding an `AZ_ENABLE_TRACE` #ifdef guard at the same time? Its usually the opposite

Reply:
The variable is actually always used on line 352, but the compiler didn't like that in a monolithic release build it was captured in a lambda but not used because the AZ_Error() function was compiled out.